overlord/hookstate/hookmgr.go

```diff
@@ -270,9 +274,27 @@ func snapCmd() string {
 	return filepath.Join(filepath.Dir(exe), "../../bin/snap")
 }
 
-func runHookAndWait(snapName string, revision snap.Revision, hookName, hookContext string, tomb *tomb.Tomb) ([]byte, error) {
+var syscallKill = syscall.Kill
+var cmdWaitTimeout = 5 * time.Second
+
+func killemAll(cmd *exec.Cmd) error {
+	pgid, err := syscall.Getpgid(cmd.Process.Pid)
```

Comment:
As @mvo5 pointed out, though, this is Getpgid which doesn't exist in the os package, and besides that there isn't much else being done. Avoiding syscall when everything else is already using it isn't a big win, so I take the comment back.